app/models/return_details.rb

```diff
@@ -0,0 +1,9 @@
+class ReturnDetails < ApplicationStruct
+  RETURN_REASONS = Types::RETURN_REASONS
+
+  attribute? :reason, Types::Nil | Types::String
+  attribute? :details, Types::Nil | Types::String
+
+  validates :details, length: { minimum: 10 }
```

Comment:
I appreciate this is sensible, but did we not agree to not have any restrictions on length of text?

Reply:
Ah perhaps I misunderstood the ticket. I thought we required some details, but would not restrict the maximum number of characters.

Reply:
Aha maybe that's right and I got confused between max and min!